modules/auxiliary/fuzzers/http/http_post_strings.rb

```diff
@@ -0,0 +1,67 @@
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Fuzzer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => "HTTP POST Fuzzer",
+      'Description'    => %q{
+        This module will fuzz a http server by sending a string generated from the options set as HTTP POST data
+      },
+      'Author'         => [ 'Aamir Mir <aamir[at]aamir.info>' ],
+      'License'        => MSF_LICENSE
+      ))
+    register_options([
+      Opt::RPORT(80),
+      OptInt.new("LENGTH", [true, "The length of the string being sent", 10]),
+      OptString.new("HEADER", [true, "HTTP Header to Fuzz, Options: none, content-type, user-agent, referer, auth, host, cookie","none"]),
+      OptString.new("CHAR", [true, "The character used to build the string", "A"]),
+      OptBool.new("PATTERN", [true, "If true the string will be created using msf pattern create rather than the value of CHAR","false"]),
+    ], self.class)
+  end
+
+def run
+  print_status("Aamir's HTTP Fuzzer Metasploit module")
```

Comment:
While I do love it and think it's kind of old-school cool, it's not really a stylistic fit for an official module. Hope you understand.